pennylane_lightning/core/src/simulators/lightning_kokkos/gates/tests/Test_StateVectorKokkos_NonParam.cpp

```diff
@@ -28,6 +28,7 @@
 #include "TestHelpers.hpp"
 #include "TestHelpersWires.hpp"
 #include "Util.hpp"
+#include "iostream"
```

Comment:
Why is that required?

Reply:
It is something I forgot to clean up, I was trying to print out something.